### PR TITLE
Namespace aliases and namespaces

### PR DESCRIPTION
**Pull request by stranger195**
If there are errors in the proposed code, please fix it.

Run NamespaceDupes.php to move pages from mainspace to the new namespaces.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -76,6 +76,22 @@
...
+define( 'NS_SUPER_MARIO_WORLD_2_TALK', 1635 );
+define( 'NS_SUPER_MARIO_BROS', 1636 );
+define( 'NS_SUPER_MARIO_BROS_TALK', 1637 );
+define( 'NS_SUPER_MARIO_ADVANCE_2', 1638 );
+define( 'NS_SUPER_MARIO_ADVANCE_2_TALK', 1639 );
+define( 'NS_SUPER_MARIO_ADVANCE_3', 1640 );
+define( 'NS_SUPER_MARIO_ADVANCE_3_TALK', 1641 );
+define( 'NS_SUPER_MARIO_ADVANCE_4', 1642 );
+define( 'NS_SUPER_MARIO_ADVANCE_4_TALK', 1643 );
+NS_MINECRAFT, NS_SUPER_MARIO_LAND_2, NS_SUPER_MARIO_WORLD_2, NS_SUPER_MARIO_BROS, NS_SUPER_MARIO_ADVANCE_2,
```
Comment (Southparkfan) [inline LocalSettings.php]: Remove this line.

File: LocalSettings.php [hunk 2]
```diff
@@ -76,6 +76,22 @@
...
+NS_SUPER_MARIO_ADVANCE_3, NS_SUPER_MARIO_ADVANCE_4
```
Comment (Southparkfan) [inline LocalSettings.php]: This one as well.

**Comment by stranger195** [comment]
@Southparkfan Looks like you misunderstood the code. If I remove the define lines, then the proposed namespaces wouldn't exist. The `NS_MINECRAFT, NS_SUPER_MARIO_LAND_2, NS_SUPER_MARIO_WORLD_2, NS_SUPER_MARIO_BROS, NS_SUPER_MARIO_ADVANCE_2,
+NS_SUPER_MARIO_ADVANCE_3, NS_SUPER_MARIO_ADVANCE_4` line was because I was adding the define lines and I forgot to remove them.

**Comment by Southparkfan** [comment]
@stranger195 as you can see I only added a commented for those two lines introducting syntax errors, so I did not misunderstand anything.

**Comment by stranger195** [comment]
@Southparkfan Okay, I get it now.

**Pull request merged**